### DISCUSSION TITLE
Read only `Switch`

### DISCUSSION THREAD

**Discussion by crabvk** [mantinedev/mantine#7131] [Feature requests]
Now (Mantine 7.14.1) `Switch`es has `readOnly` property, but it doesn't work. The `disable` property can be used instead, but it makes a `Switch` value less visible.

**Comment by rtivital** [comment]
Switch component is based on input[type="checkbox"], for read only, see https://stackoverflow.com/questions/155291/can-html-checkboxes-be-set-to-readonly

**Comment by crabvk** [comment]
Thank you.
As a result I've added [global `inert` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inert) like this:
```tsx
const inert = { inert: '' as const }
<Switch  {...inert} />
```
to make `Switch` appear like readonly.
